entity-framework/toc.yml

```diff
@@ -20,6 +20,8 @@
     items:
     - name: Welcome!
       href: core/index.md
+    - name: "What's new in EF Core 5.0"
+      href: core/what-is-new/ef-core-5.0/whatsnew.md
```

Comment:
```suggestion
      href: core/what-is-new/ef-core-5.0/whatsnew
```

Reply:
Or may need `/ef/` prefix

Reply:
Not pushing back, but it seems to work as-is with the extension (and all other entries also have the extension)?

Reply:
The subtle difference is this node is just a link to another page inside docset. That page contains actual doc content. When referencing doc content .md is required to inform this is the markdown file. For link, we can use uid directly.

Reply:
I have no idea what you just said so I'll just commit it!

Reply:
It gave warning. 😞 
Need to put /ef/core/what-is-new/ef-core-5.0/whatsnew

Reply:
Done, seems to pass with no warnings